hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1832,6 +1826,10 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	if err := r.reconcileKarpenterOperator(ctx, createOrUpdate, hcluster, hcp, r.HypershiftOperatorImage, controlPlaneOperatorImage); err != nil {
```

Comment:
why are we moving this?

Reply:
I just moved it to the end, just to make sure an error here doesn't block the HC from progressing.